src/html/header.jade

```diff
@@ -0,0 +1,7 @@
+header#main-layout-header: nav.mlh-navbar
+  #main-layout-header-left.mlh-navbar-cell
+    a#main-layout-header-home-logo( href='/' )
+      svg( version='1.1', xmlns='http://www.w3.org/2000/svg', xmlns:xlink='http://www.w3.org/1999/xlink', x='0px', y='0px', width='32px', height='32px', viewbox='0 0 32 32', style='enable-background:new 0 0 32 32;', xml:space='preserve' )
+        g
+          path( d='M0,32h32V0H0V32z M29,29h-5v-5h-3v5H3V3h26V29z' )
+          path( d='M12.108,6h-2.49L5.01,17.6h2.383l1.246-3.2h4.468l1.226,3.2h2.386L12.108,6z M9.332,12.4l1.532-4.129l1.512,4.129H9.332z' )
```

Comment:
Copy/paste.

I don't think it makes a ton of sense to include a real nav header/footer if we're erroring... Also those are going to be annoying to maintain.